content/en/about/case-studies/bluecore/index.md

```diff
@@ -0,0 +1,60 @@
+
+Kubernetes offered a path to scale, and since Bluecore was already running on Google's App Engine, it seemed like migrating portions of their workflow to GKE would be a no-brainer. However, the majority of Bluecore's engineers didn't have sufficient experience with containerized applications. This could make migrating to GKE a painful transition.
+
+Fortunately, they found a solution.
+
+## The Solution: Enabling developers with the Istio service mesh
+
+"Istio enables an approach that allows you to start building right out of the box," explains Shray Kumar, a software engineer on the infrastructure team at Bluecore. "And you can make sure you are building things the right way."
+
+Without a service mesh, breaking a monolithic application into containerized services comes with a number of challenges without obvious solutions. One of these, for example, is implementing authentication and authorization. If each containerized service needs its own solution, the chances are too great that individual developers will come up with their own way to do it. This can lead to fragmented code and plenty of future headaches.
+
+Thankfully, Kumar and the infrastructure team were familiar with Istio. They worked closely with Al Delucca, principal engineer for Bluecore's data platform team, on a plan to begin implementing it.
+
+"We had the problem," Delucca explains. "But whether or not Istio was the tool to solve it, that's what we had to figure out."
+
+They discovered that Istio's feature set provided many solutions to existing challenges. Authorization was a big one. Incoming messages from partner applications needed to be authenticated. Istio could perform that authentication at the edge, which meant that each individual service didn't need to implement its own methods.
+
+"We are able to push authentication and authorization to the edge, which takes the burden of understanding those systems away from our engineers," says Delucca. "It's not so much what they do with Istio directly, it's what Istio does for them without them knowing. That's the key—the big win for us."
+
+As their engineers began breaking the monolithic application down into services, they encountered another challenge that Istio was able to solve. Tracing calls to services looked as though it would be problematic. Many of the legacy functions didn't have clear documentation about their dependencies and requirements. This meant that performance issues and remote service calls could leave developers scratching their heads. Luckily, [Istio's distributed tracing](/docs/tasks/observability/distributed-tracing/) came to the rescue. With it, developers were able to pinpoint bottlenecks and services that needed bugfixes and additional work.
```

Comment:
```suggestion
As their engineers began breaking the monolithic application down into services, they encountered another challenge that Istio was able to solve. Tracing calls to services looked as though it would be problematic. Many of the legacy functions didn't have clear documentation about their dependencies and requirements. This meant that performance issues and remote service calls could leave developers scratching their heads. Luckily, [Istio's distributed tracing](/docs/tasks/observability/distributed-tracing/) came to the rescue. With it, developers were able to pinpoint bottlenecks and services that needed bug fixes and additional work.
```